.github/workflows/test.yml

```diff
@@ -0,0 +1,112 @@
+# This is a GitHub workflow defining a set of jobs with a set of steps.
+# ref: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions
+#
+name: Tests
+
+on:
+  pull_request:
+  push:
+
+defaults:
+  run:
+    shell: bash
```

Comment:
This is a legacy of me thinking the default shell was `sh` but it is actually `bash`! Can be removed.